docs/poc-guidlines.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: Firebolt POC Guidelines
+description: Make the necessary preparations for your Firebolt PoC
+nav_order: 12
+---
+
+# Firebolt POC Guidelines
+
+## Thank you for considering Firebolt. We created this document to help both our teams work better together during the POC.
```

Comment:
Let's avoid first person and the somewhat meaningless word "better" whenever possible...
Rewrite: 
This document will help you through the steps to set up for a proof of concept of Firebolt. 

or something like that?

Reply:
Sure!